README.md

```diff
@@ -1,13 +1,13 @@
 # Google Student Training in Engineering Program
 
-This repo contains the projects you'll work on for the first half of STEP.
-This work is based on the [Google Software Product Sprint](https://g.co/softwareproductsprint) program.
+This repo contains Clayton Ferguson's portfolio and STEP projects! 
 
-To get started:
+No need for OFF!, Cutter, or Raid, there are no bugs here.
```

Comment:
Not sure if I understand this. Is it from the starter project or some movie?

Reply:
Haha, they're bug(insect) sprays.

Reply:
Gotcha!